src/main/java/core/basesyntax/service/WriterService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface WriterService {
+    void writeToFile(List<String> report, String filePath);
```

Comment:
we should not adapt abstraction to the implementation, this writer can write anything, not only reports, to any file
```suggestion
public interface WriterService {
    void writeToFile(List<String> data, String filePath);
```